src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+
+      <p className="Person__age">
+        {`I am ${person.age || null}`}
+      </p>
+
+      <p className="Person__partner">
+        {person.isMarried ? `My hasbands name is ${person.partnerName}` : `I am not married`}
```

Comment:
Let's consider that male person can have a wife, so you need to either render text with `husband's` or `wife's` word

Reply:
Also, move it to the variable, too much going on here